tests/test_pyscriptexecutor.py

```diff
@@ -0,0 +1,52 @@
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+from threading import Thread
+from typing import Dict, Any
+
+from grinder.pyscriptexecutor import PyScriptExecutor
+
+
+class HTTPTestServer:
+    def __init__(
+        self,
+        configuration: Dict[str, Any]
+    ):
+        host = configuration['host']
+        port = configuration['port']
+
+        print('3')
```

Comment:
For what?

Reply:
it just for testing(where my programm crashed) no need now !